### PR TITLE
Fix log in PersistentTopicsBase#getPartitionedTopicMetadata

### PR DESCRIPTION
**Pull request by yuruguo**
### Motivation
Fix log information in `PersistentTopicsBase#getPartitionedTopicMetadata` method and change`cluster` to `topic`.

### Documentation 
- [x] `no-need-doc`

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java [hunk 1]
```diff
@@ -3621,15 +3621,14 @@ protected OffloadProcessStatus internalOffloadStatus(boolean authoritative) {
                     validateAdminAccessForTenant(pulsar,
                             clientAppId, originalPrincipal, topicName.getTenant(), authenticationData);
                 } catch (RestException authException) {
-                    log.warn("Failed to authorize {} on cluster {}", clientAppId, topicName.toString());
+                    log.warn("Failed to authorize {} on topic {}", clientAppId, topicName, authException);
```
Comment (Jason918) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java:3624]: IMHO, we don't need log stack trace here. `authException` can be set as the cause of the following PulsarClientException.

**Review by Jason918 — APPROVED** [review]

**Review by lovehzj — APPROVED** [review]

**Comment by yuruguo** [comment]
@k2la @Technoboy- PTAL

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**